src/abacus-ts/calculators/orders.ts

```diff
@@ -0,0 +1,196 @@
+import { IndexerBestEffortOpenedStatus, IndexerOrderStatus } from '@/types/indexer/indexerApiGen';
+import { IndexerCompositeOrderObject } from '@/types/indexer/indexerManual';
+import { HeightResponse } from '@dydxprotocol/v4-client-js';
+import { mapValues, maxBy, orderBy } from 'lodash';
+
+import { NUM_PARENT_SUBACCOUNTS } from '@/constants/account';
+
+import { assertNever } from '@/lib/assertNever';
+import { getDisplayableTickerFromMarket } from '@/lib/assetUtils';
+import { mapIfPresent } from '@/lib/do';
+import { MaybeBigNumber, MustBigNumber } from '@/lib/numbers';
+
+import { mergeObjects } from '../lib/mergeObjects';
+import { OrdersData } from '../rawTypes';
+import { OrderStatus, SubaccountOrder } from '../summaryTypes';
+
+export function calculateOpenOrders(orders: SubaccountOrder[]) {
+  return orders.filter(
+    (order) => getSimpleOrderStatus(order.status ?? OrderStatus.Open) === OrderStatus.Open
```

Comment:
is it better to use a nullish coalescing op here or just allow for `undefined` in getSimpleOrderStatus since the default -> OrderStatus.Open